src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -35,13 +35,14 @@ $dark: #777 !default;
 
 // Extra theme colors.
 $accent: $red !default;
+$accent-wcag-aa-regular-text: #25828e !default;
 $warning-dark: $orange !default;
 
 $fg-color-over-dark-bg: $white !default;
 $fg-hover-color-over-dark-bg: $gray-500 !default;
 
 $theme-colors: (
-  'accent': $accent,
+  'accent': $accent-wcag-aa-regular-text,
```

Comment:
If the intention was to change the table action button color to primary, then why not using the `btn-primary` instead of `btn-accent`? If both are the same then we shouldn't keep both of them right?
https://github.com/ceph/ceph/blob/3d710cda2f130aad4278a7debf4da503b42f116e/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-actions/table-actions.component.ts#L23

Reply:
Hi @nizamial09, red buttons mostly indicate dangerous actions. @epuertat mentioned several users have pointed it out and we thought it a good idea to change them. Since the buttons are also used in forms I found changing the accent color here fixes them too. BTW we could also update the forms seperately